lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyHoverParticipant.java

```diff
@@ -59,7 +59,7 @@ public Hover onAttributeValue(IHoverRequest request, CancelChecker cancelChecker
                 stringBuilder.append(String.format("%s = %s", variable.getValue(), variableMap.get(variable.getValue())));
             }
             if (varIter.hasNext()) {
-                stringBuilder.append(System.lineSeparator());
+                stringBuilder.append("<br />");
```

Comment:
Is the blank space in the string intentional?

Reply:
It was intentional . <br /> is the XHTML compatible way. Normal System.lineSeparator() was not working in our client side hence I tried with <br /> and it works